src/components/ScreenshotCard.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Box, Image, Text } from 'grommet';
+
+const ScreenshotCard = (props) => (
```

Comment:
Instead of using `props`, deconstruct `text` & `src`, it will clean up the code so you won't need to point to props each time you need a prop.
```suggestion
const ScreenshotCard = ({text, src}) => (
```